_about/projects/wai-coop/symposium2.md

```diff
@@ -157,10 +937,8 @@ This meeting will take place on the Zoom teleconference system with real-time ca
 - Scott Hollier (University of South Australia)
 
 
-## Acknowledgements
-{:#acknowledgements}
+![An EU Project]({{ "/content-images/wai-about-wai/eu.svg" | relative_url }}){:.right.small}
 
+This research symposium was organized by the [WAI-CooP Project](https://www.w3.org/WAI/about/projects/wai-coop/), an European Commission co-funded project, and the [W3C Accessible Platform Architectures (APA) Working Group](https://www.w3.org/WAI/APA/).
```

Comment:
@iadawn - If OK, I'd like to delete "This research symposium was organized by the [WAI-CooP Project](https://www.w3.org/WAI/about/projects/wai-coop/), an European Commission co-funded project, and the [W3C Accessible Platform Architectures (APA) Working Group](https://www.w3.org/WAI/APA/)." since we now have it in the footer (we didn't before).

Sorry @leticiaseixas for the continued edits. I guess then you could delete Acknowledgements and move "Organizing Committee" back to h2. You could make "Symposium Chairs" and "Scientific Committee" ```<p><strong>...:``` so they're not in the Page Contents.